Feature/com 5241

kibo-himanshivarshney opened the pull request:
1) Implemented the templates structure for Subscription Activation , reactivation and creation.
2) Created 3 common hyper live templates for Subscription details, Item details and Subscription Summary.

kibo-prakharkhandelwal commented on templates/email/subscription-activated.hypr:
```diff
@@ -0,0 +1,23 @@
+{% extends "email/email" %}
+{% block body-content %}
+
+<!--- Header --->
+<div class="mz-curbside-email-header">  
+<img src="https://kibocommerce.com/wp-content/themes/kibo/images/logo.png" alt="logo">
+<div class="mz-curbside-header-body">
+    <p><strong>{{ siteContext.generalSettings.websiteName }}</strong></p>
+    <h3>{{ labels.curbsideOrderConfirmationtxt }}</h3>
+</div>
+</div>
```
Do we need this Header section with Kibo logo and website name? I don't see it in the mock. Confirm with Aalekh once. If yes then we'll have to add it to other templates as well.

kibo-raghu replied:
I don't think header is required here. Remove the above code

kibo-himanshivarshney replied:
removed the header part

kibo-prakharkhandelwal commented on templates/modules/common/subscription-details.hypr.live:
```diff
@@ -0,0 +1,63 @@
+        </div>
+    </div>
+    <br/></br/>
+    <div>
+        <strong>
+        {{labels.shippingMethod}}:
+        </strong> 
+        {{model.fulfillmentInfo.shippingMethodName}}    
+    </div>
```
Display this only if the shipping method is STH.

kibo-raghu replied:
@kibo-himanshivarshney You will need to look into how details are displayed in case of orders for STH and BOPIS and implement things similarly

kibo-himanshivarshney replied:
removed the header part.

kibo-himanshivarshney replied:
fixed the review comment and got it reviewed by Sagar

kibo-raghu commented on templates/modules/common/subscription-order-summary.hypr:
```diff
@@ -0,0 +1,43 @@
+
+<table class="mz-ordersummary subscription-detail">
```
Rename file name to `subscription-summary.hypr`

kibo-himanshivarshney replied:
renamed the file name.

kibo-raghu reviewed (changes requested):
Please go over comments and make changes accordingly.

kibo-himanshivarshney committed:
COM-5241- created templates for subscription activation

email template for subscription activated

created a email template for the email trigger on subscription creation and subscription activation and subscription reactivation.
